libs/react/src/lib/form/formItems.tsx

```diff
@@ -48,21 +53,39 @@ export const FormItems: React.FC<FormItemsProps> = ({ children, validate, name }
     if (type === 'checkbox') {
       inputValue = checked ? value : null
       /* eslint-disable-next-line */
-      checked ? setValues((values: Record<string, any>) => ({ ...values, [name]: value })) : setValues((values: Record<string, any>) => ({ ...values, [name]: null }))
+      checked
+        ? setValues((values: Record<string, any>) => ({
+            ...values,
+            [name]: value,
+          }))
+        : setValues((values: Record<string, any>) => ({
+            ...values,
+            [name]: null,
+          }))
     } else {
       inputValue = value
       /* eslint-disable-next-line */
       setValues((values: Record<string, any>) => ({ ...values, [name]: value }))
     }
 
-    validateInputValue({ value: inputValue as string, name, type, checked }, validate?.rules as ValidatorRules, setErrors)
+    validateInputValue(
+      { value: inputValue as string, name, type, checked },
+      validate?.rules as ValidatorRules,
+      setErrors
+    )
   }
 
   /* eslint-disable-next-line */
-  return React.cloneElement(children as any, {
+  return React.cloneElement(children as React.ReactElement, {
     validator: errors?.[name] && validate,
     name,
-    onChange,
+    onChange: (event: React.ChangeEvent<HTMLInputElement>) => {
+      if (typeof _onFormItemsChanged === 'function') {
+        _onFormItemsChanged?.(event)
```

Comment:
No need for optional chaining here, since the `if` already ensures this is a function value :)